layouts/partials/head.html

```diff
@@ -3,40 +3,31 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ .Title }}</title>
-  <meta name="author" content="{{ .Site.Author.name }}" />
-
-  {{ if .Keywords }}
-  <meta name="keywords" content="{{ delimit .Keywords ", " }}">
-  {{ else if .Site.Params.DefaultKeywords }}
-  <meta name="keywords" content="{{ delimit .Site.Params.DefaultKeywords ", " }}">	
-  {{ end }}
-
-  {{ if .Description }}
-  <meta name="description" content="{{ .Description }}">
-  {{ else if .Site.Params.DefaultDescription }}
-  <meta name="description" content="{{ .Site.Params.DefaultDescription }}">	
-  {{ end }}
+  <title>{{ .Title | markdownify | plainify }}</title>
+  <meta name="author" content="{{ .Param "author" }}" />
+  <meta name="keywords" content="{{ with .Keywords }}{{ delimit . ", " }}{{ else }}{{ delimit .Site.Params.defaultKeywords ", " }}{{ end }}">
+  {{ $description := default .Site.Params.defaultDescription .Description }}
+  <meta name="description" content="{{ $description | markdownify | plainify }}">
 
   {{ .Hugo.Generator }}
 
+  <!-- Font -->
   <link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel='stylesheet' type='text/css'>
 
-  <!-- Bootstrap and Font Awesome css -->
+  <!-- Bootstrap and Font Awesome CSS -->
```

Comment:
I agree with the headers additions or comments refactors, but it's not related with this PR.

Reply:
You're starting troll here...

Reply:
It's not trolling, it's easy to review small and focus PR's